content/rule_book/material_and_questions.md

```diff
@@ -19,215 +42,234 @@
-    - The clarification of a single answer does not meet the above requirement.
-    - The identification of a singular pronoun does not meet the above requirement.
-    - The identification of a plural pronoun does meet the above requirement.
-    - If one of the listed answers to the multiple answer's questions is a "negative" answer, this is deemed invalid as a Multiple Answer question.
-        - A "negative" answer is an answer used to contrast another answer, or does not answer the question (e.g. An answer that begins with the words or words similar to "not" or "neither".)
+> So neither the one who plants nor the one who waters is anything, but only God, who makes things grow.
 
-##### Example
+The question "Who is anything?" would require the answer:
+
+> Neither the one who plants nor the one who waters.
+
+This is an invalid MA.
 
-- Negative word: For example, 1 Corinthians 3:7 says, "So neither the one who plants nor the one who waters is anything, but only God, who makes things grow." The question, "Who is anything?" Answer: neither the one who plants nor the one who waters, is not a valid MA
-- Contrasting: For example, 1 Corinthians 5:25 says, "Everyone who competes in the games goes into strict training. They do it to get a crown that will not last, but we do it to get a crown that will last forever." The question, "A crown that will what?" Answer: Not last; last forever, is not a valid MA.
+Consider 1 Corinthians 9:25:
+
+> Everyone who competes in the games goes into strict training. They do it to get a crown that will not last, but we do it to get a crown that will last forever.
+
+The question "A crown that will what?" with an answer "Not last; last forever" is an invalid MA.
 
 #### Reference
 
-- Used to distinguish exact same words or phrases from the material.
-- The entire reference question is part of the required question + answer.
-- If a quizzer needs to provide the reference question, the provided reference question must:
-    - Be of the exact same type (CR, CVR, MACR, MACVR) as the written question.
-    - Have content AND wording is similar to the content and wording of the written question.
-    - Have the determining word or phrase of the written question.
-    - The interrogative word used is not a factor when deciding the valid type of a reference question.
+Reference question type group questions are used to distinguish between exact duplicate words or phrases from the material. The entire reference question is part of the required question and answer.
 
-##### Example
+If a quizzer needs to provide the reference question, the provided reference question must:
+
+- Be of the exact same question type (CR, CVR, MACR, MACVR, Q, Q2V, etc.) as the written question
+- Have content and wording similar to the content and wording of the written question
+- Have the determining word or phrase of the written question
+- Have the interrogative word used not be a factor when deciding the valid type of a reference question
```

Comment:
I don't think this is what I was going for at all.     Hmmm, this needs a lot of wordsmithing, my current wording is garbage: "The interrogative word used is not a factor when deciding the valid type of a reference question."      basically, we're trying to say that the chosen interrogative word (by either the question writer or a quizzer providing a reference question), is not a factor in the "is it exact duplicate word/phrase" determination.  The interrogative word CAN dictate what the answer is, which absolutely can dictate "valid type"

The Commentary makes this clear. But it's written very bad in the rulebook now.

Reply:
How should we attack this one? Slack chatter about it for a while?

Reply:
probably.

Reply:
Jeremy's is good, IMO: "Only the material is used, not the choice of interrogative word, when deciding whether a question is INT vs. CR vs. CVR."

Reply:
3943045